AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_GridAdded.py

```diff
@@ -59,6 +62,7 @@ def AtomEditorComponents_Grid_AddedToEntity():
     2) Add a Grid component to Grid entity.
     3) UNDO the entity creation and component addition.
     4) REDO the entity creation and component addition.
+    5) Grid Size changed.
```

Comment:
Numeric values after this need to be updated to match the tests below.
Looks like everything after this is still labeled starting at 5.

Reply:
fixed